package.json

```diff
@@ -7,7 +7,7 @@
   "scripts": {
     "start": "yarn codegen:socket && yarn codegen:graphql && react-app-rewired start",
     "build": "yarn codegen:socket && yarn codegen:graphql && cross-env GENERATE_SOURCEMAP=false react-app-rewired build",
-    "build:serve": "yarn build && serve -s ./build -l 5000",
+    "build:serve": "yarn build && serve -s ./build -l 5001",
```

Comment:
why change it to 5001?

Reply:
In my machine something is always running in 5000. So thought will keep a safe port. :) No other reasoning. 🐼

Reply:
Thanks @Diogomartf for finding translation